src/main/java/me/fromgate/reactions/util/item/VirtualItem.java

```diff
@@ -301,13 +293,17 @@ public void setPotionMeta(String potions) {
         this.setItemMeta(pm);
     }
 
-    @SuppressWarnings("deprecation")
     private void setSkull(String owner) {
         if (Utils.isStringEmpty(owner))
             return;
         if (this.getItemMeta() instanceof SkullMeta) {
             SkullMeta sm = (SkullMeta) this.getItemMeta();
-            sm.setOwner(owner);
+
+            PlayerProfile playerProfile = Bukkit.createProfile(UUID.randomUUID(), null);
```

Comment:
Prefer generating UUID from owner's string - `UUID.nameUUIDFromBytes(owner.getBytes())`

Reply:
It does not change anything. The whole point is the texture in parameters

Reply:
I'm pretty sure server is caching head profiles by its UUID for internal purposes - flooding those with random UUIDs is not a great idea.